src/main/webapp/mapScript.js

```diff
@@ -1,6 +1,9 @@
+// Global Variables
+API_KEY = 'AIzaSyBf5E9ymEYBv6mAi78mFBOn8oUVvO8sph4';
```

Comment:
It may be wise to not check the API key into the repo as someone could _potentially_ steal it if they were that interested.